presto-main/src/main/java/com/facebook/presto/operator/index/IndexBuildDriverFactoryProvider.java

```diff
@@ -82,7 +82,7 @@ public DriverFactory createSnapshot(int pipelineId, IndexSnapshotBuilder indexSn
                 false,
                 ImmutableList.<OperatorFactory>builder()
                         .addAll(coreOperatorFactories)
-                        .add(new PagesIndexBuilderOperatorFactory(outputOperatorId, planNodeId, indexSnapshotBuilder))
+                        .add(new PagesIndexBuilderOperatorFactory(outputOperatorId, planNodeId, indexSnapshotBuilder, "IndexBuilder"))
```

Comment:
can we declare `IndexBuilder` as static variable and use it. same for `PageBuffer`

Reply:
Done. Thanks!